### ISSUE TITLE
[BUG] Problem with delta datasets where only metadata changes

### ISSUE THREAD

**Issue by HarikalarKutusu** [common-voice/common-voice#4302]
**Describe the bug**
There is a "small" bug in delta release creation, which seems to happen in this case:
- If there are no new recordings in delta
- But some previously recorded ones get validated/reported etc

A delta release record is generated and shown on the frontend (as it should be), but if you click it, you cannot reach the file.

**To Reproduce**

1. E.g. try to download **Assamese** Delta 16.1 from [Datasets Menu](https://commonvoice.mozilla.org/en/datasets)
2. See the error (captured below)

**Expected behavior**

- A delta record and file should be created
- The delta dataset file should not include any clips file, but should include changed `.tsv` files, so in an automatic workflow those files from previous version can be overwritten with files coming from delta.

Alternative: Create delta dataset files in every case, even if no change.

**Screenshots**

![image](https://github.com/common-voice/common-voice/assets/8849617/4205b8aa-65ce-424f-bd1f-832a26887c25)

**Additional context**
This started to happen a few CV releases back, reported on chat, but not resolved yet, so I posted it as an issue.

**Comment by moz-kathyreid** [comment]
Hi @HarikalarKutusu thank you so much for flagging this. 
I tried downloading Assamese 16.1 and was able to download it OK, so wondering if this is still current?

**Comment by HarikalarKutusu** [comment]
Yes, it seems to be fixed and it works as expected. Somehow I missed that PR :/

- [x] A delta record and file should be created
- [x] The delta dataset file should not include any clips file, but should include changed .tsv files, so in an automatic workflow those files from previous version can be overwritten with files coming from delta.

I also checked it with Azerbaijani delta v17.0. No new clips but updated metadata is there.

That will save me a lot of bandwidth...
Thank you and Hooray!
Closing.

**Comment by moz-kathyreid** [comment]
Thank you so much for checking @HarikalarKutusu - as always, really appreciate all that you do.
